src/Debugger.php

```diff
@@ -122,7 +127,47 @@ protected function updateResponse(Request $request, Response $response)
      */
     protected function needToUpdateResponse(Response $response)
     {
-        return $response instanceof JsonResponse && ! $this->storage->isEmpty();
+        $isJsonResponse = $response instanceof JsonResponse || $response->headers->contains('content-type',
+                'application/json');
+
+        return $isJsonResponse && !$this->storage->isEmpty();
+    }
+
+    /**
+     * Fetches the contents of the response and parses them to an assoc array
+     *
+     * @param Response $response
+     * @return array|bool
+     */
+    protected function getResponseData(Response $response)
+    {
+        if ($response instanceof JsonResponse) {
+            /** @var $response JsonResponse */
+            return $response->getData(true) ?: [];
+        }
+
+        $content = $response->getContent();
+
+        return json_decode($content, true) ?: false;
```

Comment:
Please, add `json_last_error()` check

Reply:
The goal here is to 'try' to decode the content, if it fails, we don't need to know the reason, just that it's not a valid json content (this is just to try and guess if it's actually a response with a json content).

If there is no error we do our thing, otherwise, just let it go

Reply:
Please merge this! I'm using dingo and it didn't work until I updated `debugger.php` with this commit.

Reply:
Done )